frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/pvc.scenario.ts

```diff
@@ -49,7 +49,7 @@ describe('Test PVC creation with options.', () => {
       accessMode: VOLUME_ACCESS_MODES.RWO,
     };
     await createNewPersistentVolumeClaim(testPvc, true);
-    expect(pvcSize.getText()).toEqual('2Gi');
+    expect(pvcSize.getText()).toEqual('2 GiB');
```

Comment:
Could we have different values for different OCP versions? It would be better not to break the tests for OCP 4.5, I think.

Reply:
@ebondare https://github.com/openshift/console/blob/release-4.5/frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/pvc.scenario.ts#L52
It is already present in 4.5 release branch.